apps/preview/next/pages/showcases/MegaMenu/OpenOnClick.tsx

```diff
@@ -0,0 +1,263 @@
+            <img src={brandLogoSign.src} alt="Sf Logo" className="hidden w-10 h-10 mr-4 md:block lg:hidden" />
+            <img src={brandLogoSign.src} alt="Sf Logo" className="block w-8 h-8 mr-4 min-w-8 md:hidden" />
+          </a>
+          <nav className="flex w-full justify-between flex-nowrap" aria-label="SF Navigation">
+            <ul>
+              <li role="none">
+                <SfButton
+                  className="block !px-2 mr-auto text-white font-body bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+                  type="button"
+                  aria-haspopup="true"
+                  aria-expanded={open}
+                  slotSuffix={
+                    <>
+                      <SfIconChevronRight className="hidden rotate-90 md:inline-flex" />
+                      <SfIconMenu className="inline-flex md:hidden" />
+                    </>
+                  }
+                  variant="tertiary"
+                  onClick={() => {
+                    setOpen(true);
+                  }}
+                >
+                  <span className="hidden md:inline-flex">Categories</span>
+                </SfButton>
+                <CSSTransition
+                  ref={nodeRef}
+                  in={open}
+                  timeout={500}
+                  unmountOnExit
+                  classNames={{
+                    enter: '-translate-x-full md:opacity-0 md:translate-x-0',
+                    enterActive: 'translate-x-0 md:opacity-100 transition duration-500 ease-in-out',
+                    exitDone: 'translate-x-0 md:opacity-100',
+                    exitActive: '-translate-x-full md:opacity-0 transition duration-500 ease-in-out',
+                  }}
+                >
+                  <SfDrawer
+                    open={open}
+                    onClose={() => setOpen(false)}
+                    placement="top"
+                    className="grid grid-cols-1 gap-6 md:grid-cols-4 bg-white max-w-xs shadow-lg p-0 !fixed md:!absolute md:!top-[5rem] md:max-w-full md:p-8"
+                  >
+                    <div className="flex items-center justify-between p-4 bg-primary-700 md:hidden">
+                      <div className="flex items-center typography-text-lg font-medium text-white">Categories</div>
+                      <SfButton
+                        square
+                        variant="tertiary"
+                        aria-label="Close drawer"
+                        onClick={() => {
+                          setOpen(!open);
+                        }}
+                        className="text-white"
+                      >
+                        <SfIconClose />
+                      </SfButton>
+                    </div>
+                    {categoriesContent.map(({ heading, items }) => (
+                      <div className="flex-1/4" key={heading}>
```

Comment:
`flex-1/4` does not exists, your `div` just works because then split into 25% just because there is 4 columns, this class can be removed if we want to have columns fluidly split width, but this might lead into unequal columns when one will have longer unbreakable text, so I suggest use `basis-1/4` and test/add break word on whole container
![image](https://user-images.githubusercontent.com/2566152/230635528-b94ab949-1433-4b6f-9d57-95ec4c8d9834.png)

Reply:
@Szymon-dziewonski actually the flex-1/4 is reduntant here as we use grid, that is some leftover so I'll remove it